tools/run_tests/run_xds_tests.py

```diff
@@ -84,6 +84,9 @@
     'traffic_splitting',
     'path_matching',
     'header_matching',
+    'api_listener',
```

Comment:
Is there a specification for the new tests? Would be helpful to add to https://github.com/grpc/grpc/blob/master/doc/xds-test-descriptions.md

Reply:
didn't realize this one, will definitely add there, probably in a separate PR.